app/channels/client_channel.rb

```diff
@@ -0,0 +1,19 @@
+class ClientChannel < ApplicationCable::Channel
+  def subscribed
+    begin
+      @client = Client.find(params[:id])
+    rescue ActiveRecord::RecordNotFound
+      reject
+    end
+
+    unless current_ability.can? :read, @client
+      reject
+    end
+
+    stream_for @client
+  end
+
+  def self.broadcast_contact_record(contact_record)
+    broadcast_to(contact_record.client, [".message-list", ApplicationController.render(partial: 'shared/message_list_contact_record', locals: { contact_record: contact_record })])
```

Comment:
why does the backend need to know this CSS class? Couldn't the front end figure this out on its own?

Reply:
If .message-list is a class on the notes page (i think we used the same class on both pages), will this also append new client contacts (text and emails) to the notes page by accident?

Reply:
@bytheway875 It looks like we're [scoping the subscription](https://github.com/codeforamerica/vita-min/blob/master/app/javascript/listeners/index.js#L11) to the messages page, so even though the javascript is included on all pages, the callback wouldn't be triggered.